### PR TITLE
Wait tfs before starting gunicorn

### PR DESCRIPTION
**Pull request by liangma8712**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON Checklist
* When creating a PR:
- [ ] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above
#### Benchmark Checklist
* When creating a PR:
- [ ] I've modified `src/config/test_config.py` in my PR branch by setting `ENABLE_BENCHMARK_DEV_MODE = True`
* When PR is reviewed and ready to be merged:
- [ ] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Sync PR from https://github.com/aws/sagemaker-tensorflow-serving-container/pull/192/files
*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py [hunk 1]
```diff
@@ -235,4 +241,29 @@ def __init__(self, key, env_var, value, defaulted_message):
...
+                if response.status_code == 200:
+                    versions = json.loads(response.content)["model_version_status"]
+                    if all(version["state"] == "AVAILABLE" for version in versions):
+                        break
+            except (ConnectionRefusedError, NewConnectionError,
+                    MaxRetryError, requests.exceptions.ConnectionError):
+                log.warning("model: {} is not available yet ".format(tfs_url))
+                time.sleep(wait_interval_seconds)
+
+    log.info("model: {} is available now".format(tfs_url))
```
Comment (mitroviv) [inline tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py:269]: do we need a new line at the end of the file?

**Review by mitroviv — APPROVED** [review]

File: tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py (hunk 1 above)
Comment (liangma8712) [reply tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py:269]: Good question.  I just kept the same style as the earlier version.

File: src/config/build_config.py [hunk 2]
```diff
@@ -5,9 +5,9 @@
 # Do remember to revert it back to False before merging any PR (including NEURON dedicated PR)
 ENABLE_NEURON_MODE = False
 # Frameworks for which you want to disable both builds and tests
-DISABLE_FRAMEWORK_TESTS = []
+DISABLE_FRAMEWORK_TESTS = ["mxnet", "pytorch"]
```
Comment (tejaschumbalkar) [inline src/config/build_config.py:8]: To be reverted before merge

File: tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py (hunk 1 above)
Comment (tejaschumbalkar) [inline tensorflow/inference/docker/build_artifacts/sagemaker/tfs_utils.py:269]: Please add a empty line

**Review by tejaschumbalkar — APPROVED** [review]

**Pull request merged**